app/models/concerns/reassociatable.rb

```diff
@@ -7,16 +7,18 @@ module Reassociatable
   BLANK_VALUE = "_blank_"
 
   # triggers the reassociate process
-  def reassociate_child_oids
+  def reassociate_child_oids(start_index = 0)
     return unless batch_action == "reassociate child oids"
-    parents_needing_update, parent_destination_map = update_child_objects
-    update_related_parent_objects(parents_needing_update, parent_destination_map)
+    parents_needing_update, parent_destination_map, denote_continue = update_child_objects(start_index)
+    update_related_parent_objects(parents_needing_update, parent_destination_map, denote_continue)
```

Comment:
You don't need to pass denote_continue to the update_related_parent_objects().
You can just return it from this method.  update_child_object() keeps track of where it is in the list. update_related_parent_objects() doesn't care about that info.